batch/batch/globals.py

```diff
@@ -17,4 +17,4 @@
 HTTP_CLIENT_MAX_SIZE = 8 * 1024 * 1024
 
 BATCH_FORMAT_VERSION = 2
-INSTANCE_VERSION = 2
+INSTANCE_VERSION = 'v2'
```

Comment:
Ah, is this OK?  From the database:

 >   `version` INT NOT NULL,

I see, this isn't really used.  I'd say either (1) don't change it, or (2) bump it to 3 because the interface did change (the schema of the status return value is different).

Reply:
I changed it to 3.